apps/ensindexer/ponder/src/register-handlers.ts

```diff
@@ -0,0 +1,51 @@
+/**
+ * This file is loaded by ponder as part of ponder dev & ponder start. In here we conditionally
```

Comment:
here is the new register-handlers.ts that aggregates the individual event-handlers.ts files, simplifying the understanding of how ponder expects handlers to be registered within the `(root)/src` directory